networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/GossipFailureLogger.java

```diff
@@ -47,8 +47,14 @@ public synchronized void logWithSuppression(final Throwable error, final UInt64
           "Failed to publish {}(s) for slot {} because the message has already been seen",
           messageType,
           lastErroredSlot);
-    } else if (lastRootCause instanceof NoPeersForOutboundMessageException
-        || lastRootCause instanceof SemiDuplexNoOutboundStreamException) {
+    } else if (lastRootCause instanceof NoPeersForOutboundMessageException) {
+      LOG.log(
+          suppress ? Level.DEBUG : Level.WARN,
+          "Failed to publish {}(s) for slot {}; {}",
+          messageType,
+          lastErroredSlot,
+          rootCause.getMessage());
+    } else if (lastRootCause instanceof SemiDuplexNoOutboundStreamException) {
       LOG.log(
           suppress ? Level.DEBUG : Level.WARN,
           "Failed to publish {}(s) for slot {} because no peers were available on the required gossip topic",
```

Comment:
this seems to me the message for `NoPeersForOutboundMessageException`. And the above seems a more generic one so matching `SemiDuplexNoOutboundStreamException`